docs/source/docs/get-started/quick-start-guide/examples.rst

```diff
@@ -0,0 +1,98 @@
+Examples
+########
+
+- Convert PASCAL VOC dataset to COCO format, keep only images with ``cat`` class
+  presented:
+
+.. code-block::
+
+    # Download VOC dataset:
+    # http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar
+    datum convert --input-format voc --input-path <path/to/voc> \
+                  --output-format coco \
+                  --filter '/item[annotation/label="cat"]' \
+                  -- --reindex 1 # avoid annotation id conflicts
+
+- Convert only non-``occluded`` annotations from a
+  `CVAT <https://github.com/openvinotoolkit/cvat>`_ project to TFrecord:
+
+.. code-block::
+
+    # export Datumaro dataset in CVAT UI, extract somewhere, go to the project dir
+    datum filter -e '/item/annotation[occluded="False"]' --mode items+anno
+    datum export --format tf_detection_api -- --save-images
+
+- Annotate MS COCO dataset, extract image subset, re-annotate it in
+  `CVAT <https://github.com/openvinotoolkit/cvat>`_, update old dataset:
```

Comment:
we may be possible to remove re-annotate part in CVAT

Reply:
Is there a reason you want to delete the re-annotate part? It explains how to update the new annotation in Datumaro, and I think it's good to have one. e.g., datum project update reannotation-upd